src/jit/emitxarch.cpp

```diff
@@ -1494,16 +1494,15 @@ inline UNATIVE_OFFSET emitter::emitInsSizeRR(instruction ins, regNumber reg1, re
     emitAttr size = EA_SIZE(attr);
 
     UNATIVE_OFFSET sz;
-#ifdef _TARGET_AMD64_
-    // If Byte 4 (which is 0xFF00) is non-zero, that's where the RM encoding goes.
+
+    // If Byte 4 (which is 0xFF00) is zero, that's where the RM encoding goes.
```

Comment:
Oddly, I believe that the previous comment is actually correct, if confusing. The encoding for some reason puts byte 4 in byte 3 (though I would probably have zero-based the counting and called it byte3. Anyway, see instrsxarch.h near line 139 where there's a comment that talks about this.